service/src/test/java/bio/terra/workspace/db/ResourceDaoTest.java

```diff
@@ -587,6 +595,28 @@ public void listControlledResourceWithoutRegion() {
     assertEquals(
         15, resourceDao.listControlledResourcesWithMissingRegion(CloudPlatform.GCP).size());
     assertTrue(resourceDao.listControlledResourcesWithMissingRegion(CloudPlatform.AZURE).isEmpty());
+    resourceDao.deleteAllControlledResources(workspaceUuid, CloudPlatform.GCP);
+    resourceDao.deleteAllControlledResources(workspaceUuid2, CloudPlatform.GCP);
+    resourceDao.deleteAllControlledResources(workspaceUuid3, CloudPlatform.GCP);
+  }
+
+  @Test
+  public void listControlledBigQueryDatasetsWithoutLifetime() {
+    UUID workspaceUuid = createWorkspaceWithGcpContext(workspaceDao);
+    for (int i = 1; i <= 6; i++) {
+      ControlledBigQueryDatasetResource dataset =
+          ControlledResourceFixtures.makeDefaultControlledBqDatasetBuilder(workspaceUuid)
```

Comment:
what about you just do
for (int i = 0; i < 5; i++) {
   ControlledBigQueryDatasetResource dataset =
          ControlledResourceFixtures.makeDefaultControlledBqDatasetBuilder(workspaceUuid)
                .defaultTableLifetime(DEFAULT_CREATED_BIG_QUERY_TABLE_LIFETIME)
                .defaultPartitionLifetime(DEFAULT_CREATED_BIG_QUERY_PARTITION_LIFETIME)
                .build();
       resourceDao.createControlledResource(dataset);
}

ControlledBigQueryDatasetResource datasetWithoutLifetime =
          ControlledResourceFixtures.makeDefaultControlledBqDatasetBuilder(workspaceUuid)
              .defaultTableLifetime(null)
              .defaultPartitionLifetime(null)
              .build();
              
     resourceDao.createControlledResource(datasetWithoutLifeTime);

assertEquals(5, ans);